app/views/products/qhp/summary.html.erb

```diff
@@ -1 +1,191 @@
-<%= render partial: "shared/summary", locals: {qhp: @qhp} %>
\ No newline at end of file
+<% @qhp ||= qhp %>
+<% @hbx_enrollment ||= hbx_enrollment %> 
+<% @plan ||= @hbx_enrollment&.plan  %>
+
+<% if !hbx_enrollment.shopping? %>
+<%= link_to '<i class="fa fa-chevron-left"></i> Back to My Account'.html_safe, family_account_path, class: 'btn btn-default all-plans' %>
+<br/>
+<br/>
+<% end %>
+<% if qhp.present? %>
```

Comment:
I'm not sure how this condition will ever be true. Back when this was in a partial `qhp` without an `@` could have been passed as a local. But this is in a regular view. `qhp` would have to be set at the top or made an instance variable. It could even be created as an local variable from an instance variable. But this will never be true. Let's change all of these to instance variables and not set any variable up at the top of this view.